2019/2019-potential-lectures-volunteer-list.md

```diff
@@ -4,4 +4,8 @@
 - CSS layout with flexbox - Lennyroy
 - Accessibility - Christina
 - Persisting data, databases - Christina
-- Making games with Scratch - Christina
+- ~Making games with Scratch - Christina~ (Malik gave this amazing talk!)
+- Terminal - Mark
+- Github - Mark
+- JavaScript deep dive - Elana
```

Comment:
Could we reword this to maybe something more like "Fundamental programming concepts (variables, conditionals, loops)"? I'll use JS for the examples, but I don't actually know that much about JS so I don't want it to sound like I'm going to be diving into the inner workings of the language specifically. 🌚